exchanges/gateio/gateio_wrapper.go

```diff
@@ -180,6 +180,8 @@ func (g *Gateio) SetDefaults() {
 	g.WebsocketResponseMaxLimit = exchange.DefaultWebsocketResponseMaxLimit
 	g.WebsocketResponseCheckTimeout = exchange.DefaultWebsocketResponseCheckTimeout
 	g.WebsocketOrderbookBufferLimit = exchange.DefaultWebsocketOrderbookBufferLimit
+
```

Comment:
```suggestion
```

Reply:
[14258f8](https://github.com/thrasher-corp/gocryptotrader/pull/1863/commits/14258f88b55aa9861a170843c13908b1d604a34e)

Nits should be done now; thanks!